ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, triage
+assignees: ''
+
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!-- If you're describing a bug, tell us what should happen
+     If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!-- If describing a bug, tell us what happens instead of the expected behavior
+     If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!-- Not obligatory, but suggest a fix/reason for the bug,
+     or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!-- Provide a link to a live example, or an unambiguous set of steps
+     to reproduce this bug. -->
+
+1.
+
+## Context
+<!-- How has this issue affected you? What are you trying to accomplish?
+     Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Regression
+<!-- Is this issue a regression? (Yes / No)
+     If Yes, optionally please include version or commit id or PR# that caused this regression,
+     if you have these details -->
+
+## Your Environment
+<!-- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
+* Server setup and configuration:
+* Operating System and version (`uname -a`):
```

Comment:
Add a `labels` section with recommendation to add relevant labels? It can have project-specific labels, but should be sufficient as a general reminder for this template.

Reply:
See neofs-node templates for example.